### PR TITLE
Add CMintS

### PR DESCRIPTION
**Pull request by Manvel**
I hope I did everything right, hope to see CMintS being added into the list as well :)

Kudos to staticgen, awesome project BTW!

### CODE REVIEW

**Comment by Manvel** [comment]
Seems like the builds are in general broken according to https://github.com/netlify/staticgen/issues/400 ?

File: content/projects/cmints.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+repo: Manvel/cmints
+homepage: https://cmints.io/
+language:
+  - JavaScript
+license:
+  - MIT
+templates:
+  - EJS
+description: CMS and Static Content Generator created with the internationalization in mind.
+startertemplaterepo: Manvel/cmints-website
```
Comment (erquhart) [inline content/projects/cmints.md]: This repo needs to be deployable on Netlify, so you'll need a `package.json` on the repo with a `build` command that outputs a production site. If a repo that works that way isn't available, you can always remove `startertemplaterepo` for now and add one later.

Comment (Manvel) [reply content/projects/cmints.md]: Done, I have removed the `startertemplaterepo` for now, will see how I can make this compatible with Netlify and update accordingly later on. 
Thanks.

**Comment by Manvel** [comment]
@erquhart thanks for the comment, done, can't wait to see it published. Will check later check how I can make it compatible with Netlify.

**Pull request merged**